src/cli.ts

```diff
@@ -1,51 +1,537 @@
+      group: 'App Creation Options',
+      type: 'boolean',
+    })
+    .option('upgrade', {
+      describe:
+        'Upgrade an app built by an older version of Nativefier.\nYou must pass the full path to the existing app executable (app will be overwritten with upgraded version by default)',
+      group: 'App Creation Options',
+      normalize: true,
+      type: 'string',
+    })
+    .option('widevine', {
+      default: false,
+      description:
+        "use a Widevine-enabled version of Electron for DRM playback (use at your own risk, it's unofficial, provided by CastLabs)",
+      group: 'App Creation Options',
+      type: 'boolean',
+    })
+    // App Window Options
+    .option('always-on-top', {
+      default: false,
+      description: 'enable always on top window',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('background-color', {
+      description:
+        "sets the app background color, for better integration while the app is loading. Example value: '#2e2c29'",
+      group: 'App Window Options',
+      type: 'string',
+    })
+    .option('bookmarks-menu', {
+      description: 'Path to JSON configuration file for the bookmarks menu',
+      normalize: true,
+      group: 'App Window Options',
+      type: 'string',
+    })
+    .option('browserwindow-options', {
+      coerce: parseJson,
+      description:
+        'a JSON string that will be sent directly into electron BrowserWindow options. See https://github.com/nativefier/nativefier/blob/master/docs/api.md#browserwindow-options',
+      group: 'App Window Options',
+      type: 'string',
+    })
+    .option('disable-context-menu', {
+      default: false,
+      description: 'disable the context menu (right click)',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('disable-dev-tools', {
+      default: false,
+      description: 'disable developer tools (Ctrl+Shift+I / F12)',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('full-screen', {
+      default: false,
+      description: 'always start the app full screen',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('height', {
+      defaultDescription: '800',
+      description: 'set window default height in pixels',
+      group: 'App Window Options',
+      type: 'number',
+    })
+    .option('hide-window-frame', {
+      default: false,
+      description: 'disable window frame and controls',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('m', {
+      alias: 'show-menu-bar',
+      default: false,
+      description: 'set menu bar visible',
+      group: 'App Window Options',
+      type: 'boolean',
+    })
+    .option('max-height', {
+      defaultDescription: 'unlimited',
+      description: 'set window maximum height; in pixels',
```

Comment:
Nit: consistency with the other options ending with "in pixels"

```suggestion
      description: 'set window maximum height in pixels',
```